Signal/src/call/CallAudioService.swift

```diff
@@ -321,8 +321,7 @@ import AVFoundation
                 let oldMode = session.mode
                 let oldOptions = session.categoryOptions
 
-                if oldCategory == category, oldMode == mode, oldOptions == options {
-                    Logger.debug("\(self.TAG) in \(#function) doing nothing, since audio session is unchanged.")
+                guard oldCategory != category || oldMode != mode || oldOptions != options else {
```

Comment:
Don't you want to also make the analogous change below in the other "doing nothing, since audio session is unchanged" check?